runtime/vara/src/migrations.rs

```diff
@@ -24,4 +24,6 @@ pub type Migrations = (
     pallet_gear_program::migration::AppendStackEndMigration<Runtime>,
     // migration for removed waiting init list
     pallet_gear::migrations::MigrateWaitingInitList<Runtime>,
+    pallet_nomination_pools::migration::versioned_migrations::V5toV6<Runtime>,
+    pallet_nomination_pools::migration::versioned_migrations::V6ToV7<Runtime>,
```

Comment:
I believe, the `try_runtime` test will confirm the consistency of two migrations applied to the same pallet back to back, will it?

Reply:
Yes. Back to back is no problem. But state checking was disabled for older versions in our fork. The polkadot-sdk team does not use state checks and missed this case.
https://github.com/gear-tech/polkadot-sdk/pull/2